stdlib/3/subprocess.pyi

```diff
@@ -4,21 +4,61 @@
 
 from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union
 
-# TODO force keyword arguments
-# TODO more keyword arguments
-def call(args: Union[str, Sequence[str]], *, stdin: Any = ..., stdout: Any = ...,
-         stderr: Any = ..., shell: bool = ...,
+# Same args as Popen.__init__
+def call(args: Union[str, Sequence[str]],
```

Comment:
ah, didn't know about the new keyword-only-args feature -- but looks like the Popen in python3 isn't using it